agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -511,7 +511,6 @@ bool backhaul_manager::socket_disconnected(Socket *sd)
                     LOG(ERROR) << "cmdu creation of type TOPOLOGY_NOTIFICATION_MESSAGE, has failed";
                     return false;
                 }
-                cmdu_header->flags().relay_indicator = true;
```

Comment:
I think this should be fixed before the PR can be merged, no?

Reply:
no :(